components/tabs/PublicUsersTabs.tsx

```diff
@@ -6,69 +6,48 @@ import { usePathname } from 'next/navigation';
 
 const tabs = [
   {
-    name: 'ADMINS',
-    href: 'admins',
+    name: 'Admins',
+    href: 'Admins',
   },
   {
-    name: 'BILLING VIEWERS',
-    href: 'billing-viewers',
+    name: 'Billing Viewers',
+    href: '/BillingViewers',
   },
   {
-    name: 'DEVELOPERS',
-    href: 'developers',
+    name: 'Developers',
+    href: '/Developers',
   },
   {
-    name: 'SECURITY AUDITORS',
-    href: 'security-auditors',
+    name: 'Security Auditors',
+    href: '/SecurityAuditors',
   },
   {
-    name: 'VIEWERS',
-    href: 'viewers',
+    name: 'Viewers',
+    href: '/Viewers',
   },
 ];
 
-export default function Tabs({ className }: { className?: string }) {
+export default function PublicUsersTabs() {
   const pathname = usePathname();
-
   return (
-    <div className="w-full">
-      {/* <div className="md:hidden">
-        <label htmlFor="tabs" className="sr-only">
-          Select a tab
-        </label>
-        <select
-          id="tabs"
-          name="tabs"
-          className="font-bcsans text-xl block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
-          defaultValue={tabs[0]?.name}
-        >
-          {tabs.map((tab) => (
-            <option key={tab.name}>{tab.name}</option>
-          ))}
-        </select>
-      </div>
-      <div className="hidden md:block  justify-start">
-        <div className="border-b border-gray-200">
-          <div className="w-full -mb-px flex justify-between items-center" aria-label="Tabs">
-            <div className=" -mb-px flex justify-start">
-              {tabs.map((tab) => (
-                <Link
-                  key={tab.name}
-                  href={`/${pathname}/${tab.href}`}
-                  className={classNames(
-                    pathname.split('/').indexOf(tab.href) !== -1
-                      ? "relative border-bcorange text-bcblue before:content-[''] before:absolute before:w-2/4 before:border-b-3 before:border-bcorange before:bottom-0 before:left-1/2 before:-translate-x-1/2"
-                      : "relative border-transparent text-gray-300 hover:before:content-[''] hover:before:absolute hover:before:w-2/4 hover:before:border-b-3 hover:before:border-gray-300 hover:before:bottom-0 hover:before:left-1/2 hover:before:-translate-x-1/2",
-                    'lg:ml-20 w-50 py-5 text-center font-bcsans text-lg font-bold',
-                  )}
-                >
-                  {tab.name}
-                </Link>
-              ))}
-            </div>
-          </div>
-        </div>
-      </div> */}
+    <div>
+      <span className="isolate inline-flex rounded-md shadow-sm py-2 px-4">
+        {tabs.map((tab, index) => (
+          <Link
+            key={index}
+            href={`${pathname.split('/').splice(0, 5).join('/')}/${tab.href}`}
+            type="button"
+            className={classNames(
+              (pathname.split('/').includes(tab.href) ? 'bg-gray-200 hover:none' : 'bg-white hover:bg-gray-100') +
```

Comment:
I am concerns of its reliability of the check; let's explicitly check the position of the segments as well.
We could also make it as a helper function.
```js
function hasSegmentAtPosition(pathname, targetSegment, position) {
    const pathSegments = pathname.split('/');
    
    // Check if the specified position is within the array bounds
    if (position >= 0 && position < pathSegments.length) {
        return pathSegments[position] === targetSegment;
    }
    
    // If the position is out of bounds, return false
    return false;
}
```